src/test/java/systemtests/ClearCommandSystemTest.java

```diff
@@ -34,6 +34,7 @@ public void clear() {
         selectBook(Index.fromOneBased(1));
         assertCommandSuccess(ClearCommand.COMMAND_WORD);
         assertSelectedBookListCardDeselected();
+        assertBookDetailsPanelUnchanged();
```

Comment:
supposed to change right? like not show the selection anymore.

Reply:
Keeping the behavior unchanged for now. Can change this in a later PR.